File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/metnorway/locationforecast/timeseries/Instant.java
package com.lifedawn.bestweather.data.remote.retrofit.responses.metnorway.locationforecast.timeseries;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Instant {
	@Expose
	@SerializedName("details")
	private Details details;
	
	public Details getDetails() {
		return details;
	}
	
	public void setDetails(Details details) {
		this.details = details;
	}
}

File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/metnorway/locationforecast/timeseries/Instant.kt
package com.lifedawn.bestweather.data.remote.retrofit.responses.metnorway.locationforecast.timeseries

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Instant {
    @Expose @SerializedName("details") var details: Details? = null
}